chapter03/qli/src/main/java/spring/training/model/Account.java

```diff
@@ -0,0 +1,111 @@
+package spring.training.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Account {
+	private int accountNumber;
+	private String accountDescription;
+	private double accountAmount;
+	private Customer customer;
+	
+	public final static double MAX_AMOUNT = 100000000;
+
+	private List<AccountTranscation> transactions = new ArrayList<>();
+	
+	
+	public int getAccountNumber() {
+		return accountNumber;
+	}
+
+	public Account setAccountNumber(int accountNumber) {
+		this.accountNumber = accountNumber;
+		return this;
```

Comment:
Actually this is really just a way to write, but I'm not sure if this is inline with Spring underline processing, my Junit test passes, so I guess it works. 

Back to the `return this`, it's called [Method Chaining](https://en.wikipedia.org/wiki/Method_chaining), many people have been pushing for these to be [default behavior for void method since Java7](http://tech.puredanger.com/java7#chained). 

So I don't have to write, 
```java
Account acc = new Account();
acc.setAccountNumber(00000001);
acc.setAccountDescription("test account 2")
...
```
Since I'm still a bit of anti-constructor :smile: and try to work with properties, but that's probably a separate topics again. lol